pkg/webhook/validatingwebhook.go

```diff
@@ -173,6 +172,19 @@ func (m *multiClusterHubValidator) validateDelete(req admission.Request) error {
 		}
 	}
 
+	discoveryConfigList := &unstructured.UnstructuredList{}
+	discoveryConfigList.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "discovery.open-cluster-management.io",
+		Version: "v1",
+		Kind:    "DiscoveryConfigList",
+	})
+
+	discoveryConfigErr := m.client.List(context.TODO(), discoveryConfigList)
```

Comment:
Does this mean we expected DiscoveryConfigs across all namespaces to be deleted before deleting MCH?

Reply:
Yes, this checks for discoveryconfigs across all namespaces